src/Akka.Streams.Kafka/Akka.Streams.Kafka.csproj

```diff
@@ -5,6 +5,7 @@
     <Description>Apache Kafka adapter for Akka.NET Streams</Description>
     <TargetFramework>$(NetStandardLibVersion)</TargetFramework>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+      <LangVersion>9.0</LangVersion>
```

Comment:
Can't use 9.0 without upgrading the runtime I think.

Reply:
Downgraded to 8.0